eng/pipelines/templates/jobs/perf.yml

```diff
@@ -36,6 +36,7 @@ resources:
     type: github
     endpoint: Azure
     name: Azure/azure-sdk-tools
+    ref: main
```

Comment:
At some point we will want this to use our tools version tags instead of main to pin our tools so things stay functioning and there is an explicit move forward with updates.